crates/bevy_pbr/src/render/light.rs

```diff
@@ -1110,12 +1110,12 @@ pub fn prepare_lights(
             .take(spot_light_shadow_maps_count)
             .enumerate()
         {
-            let spot_view_matrix = spot_light_view_matrix(&light.transform);
+            let spot_view_matrix = spot_light_view_from_world(&light.transform);
```

Comment:
This gets assigned to world from view and I think the spot light view matrix function creates a view matrix in bevy’s current terms which means world from view.